packages/fxa-settings/src/components/Settings/PageRecoveryPhoneSetup/index.tsx

```diff
@@ -57,17 +51,22 @@ export const PageRecoveryPhoneSetup = ({
   };
 
   const sendCode = async () => {
-    // Placeholder until we have a proper SMS code sender
+    // Just retry adding the number and another code will send. Note that more than
+    // one code can be valid at the same time if the user clicks “resend code” to
+    // account for SMS transmission delay.
+    // try/catch is in the component that calls this function
```

Comment:
Noting that @vbudhram recommends invalidating the previous code when a new one is sent one. There's a thread up for discussion in the Slack channel for this project

Reply:
Was the try-catch comment meant to stay in, or a self-review note?

Reply:
Sure - sounds like we have a separate issue filed for that though.

I left that in because at first I wrapped this in the `try/catch` since usually our `account.whatever` methods have a try/catch, and then realized we didn't need it. Can remove if you want though.